apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -204,10 +205,23 @@ export class PnpmOptionsConfiguration {
    */
   public readonly resolutionStrategy: ResolutionStrategy;
 
+  /**
+   * If true, then rush will block "rush install" when the lockfile has been manually edited.
+   *
```

Comment:
Today Rush always refers generically to the various package manager lock files as a "shrinkwrap file" because this was the original NPM terminology. But now all three package managers have started calling them "lock files", so it would make sense to update Rush's terminology.  But I think we'd want to do this all at once as a breaking change.

It's confusing to call it a "lockfile" in just one place, when we say "shrinkwrap" file elsewhere in the same source file.